tests/foreman/cli/test_location.py

```diff
@@ -45,8 +35,10 @@ def test_create_location_with_different_names_positive(self, name):
         @Assert: Location is created successfully and has proper name
 
         """
-        loc = make_location({'name': name})
-        self.assertEqual(loc['name'], name)
+        for name in valid_data_list():
```

Comment:
That statement will periodically fail as Location maximum length is 246 symbols, not 255 as defined in `valid_data_list()`.
![image](https://cloud.githubusercontent.com/assets/10895065/10262628/05af80aa-69d9-11e5-9aec-e411fab138cd.png)
Actually, that data decorator had 255 symbols, but was fixed due some failures on Jenkins.
Please, fix